src/Workspaces/Core/Portable/Rename/Renamer.RenameDocumentAction.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System.Collections.Immutable;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.Rename
+{
+    public static partial class Renamer
+    {
+        /// <summary>
+        /// Individual action from RenameDocument APIs in <see cref="Renamer"/>.
```

Comment:
can you flesh this out?  i.e. the name is mostly clear, but it would be nice to have a human readable description about waht this action will be doing.

Reply:
Added more information.